{
    "has_execution_error": true,
    "execution_output": "Fatal Python error: init_sys_streams: can't initialize sys standard streams\nPython runtime state: core initialized\nOSError: [Errno 9] Bad file descriptor\n\nCurrent thread 0x00000001dcac5c40 (most recent call first):\n  <no Python frame>\n",
    "pred": {
        "error": "malformed_output"
    },
    "gold": {
        "itinerary": [
            {
                "action": "meet",
                "person": "Jessica",
                "start_time": "09:13",
                "end_time": "11:13"
            },
            {
                "action": "meet",
                "person": "John",
                "start_time": "11:18",
                "end_time": "11:33"
            },
            {
                "action": "meet",
                "person": "Nancy",
                "start_time": "11:40",
                "end_time": "12:25"
            },
            {
                "action": "meet",
                "person": "Rebecca",
                "start_time": "12:49",
                "end_time": "14:04"
            },
            {
                "action": "meet",
                "person": "Jason",
                "start_time": "15:15",
                "end_time": "17:15"
            },
            {
                "action": "meet",
                "person": "Sarah",
                "start_time": "17:30",
                "end_time": "18:15"
            },
            {
                "action": "meet",
                "person": "Mark",
                "start_time": "18:28",
                "end_time": "19:58"
            },
            {
                "action": "meet",
                "person": "Kevin",
                "start_time": "20:45",
                "end_time": "21:45"
            }
        ]
    },
    "status": "Execution error: malformed_output",
    "violated_constraint": {},
    "is_exact_match": false,
    "constraints_satisfied": false,
    "pass_number": 4
}